PlaidLink/Classes/PLDLinkBankMFAContainerViewController.h

```diff
@@ -24,6 +24,11 @@
 
 @property(nonatomic, weak) id<PLDLinkBankMFAContainerViewControllerDelegate> delegate;
 
-- (instancetype)initWithInstitution:(PLDInstitution *)institution product:(PlaidProduct)product;
+- (instancetype)initWithInstitution:(PLDInstitution *)institution
+                            product:(PlaidProduct)product;
+
+- (instancetype)initWithInstitution:(PLDInstitution *)institution
```

Comment:
Please remove this initializer. The API will be cleaner if you make options an NSDictionary that is a property